Library/Formula/pillar.rb

```diff
@@ -0,0 +1,25 @@
+class Pillar < Formula
+  homepage "https://github.com/comeara/pillar"
+  url "https://github.com/comeara/pillar/archive/v2.0.1.tar.gz"
+  version "2.0.1"
```

Comment:
This'll be redundant. `brew` can read versions automatically providing the URL is relatively simple, which it is here.